test/client/partialPayments.ts

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable @typescript-eslint/no-explicit-any -- required for formatting transactions */
+import { expect } from 'chai'
+
+import type { TransactionStream } from '../../src'
+import rippled from '../fixtures/rippled'
+import { setupClient, teardownClient } from '../setupClient'
+
+const partialPaymentIOU = rippled.partial_payments.iou
+const partialPaymentXRP = rippled.partial_payments.xrp
+
+describe('client handling of tfPartialPayments', function () {
+  beforeEach(setupClient)
+  afterEach(teardownClient)
+
+  it('Tx with no tfPartialPayment', async function () {
+    this.mockRippled.addResponse('tx', rippled.tx.Payment)
+    const resp = await this.client.request({ command: 'tx' })
+
+    expect(resp.warnings).to.equal(undefined)
+  })
+
+  it('Tx with IOU tfPartialPayment', async function () {
+    const mockResponse = { ...rippled.tx.Payment, result: partialPaymentIOU }
```

Comment:
Why not just copy the `...rippled.tx.Payment` parts of the fixture into the` partialPaymentIOU` fixture?

Reply:
Because I wanted to use the same Partial Payment to test all three responses.